srcpkgs/msp430-toolchain/patches/README-build.patch

```diff
@@ -0,0 +1,105 @@
+--- a/msp430-gcc-9.3.1.11-source-full/README-build.sh
++++ b/msp430-gcc-9.3.1.11-source-full/README-build.sh
+@@ -28,14 +28,13 @@
+ set -e
+ set -x
+ 
+-configure_args_common='--target=msp430-elf --enable-languages=c,c++ --disable-nls --enable-initfini-array'
++configure_args_common='--target=msp430-elf --enable-languages=c --disable-nls --enable-initfini-array --prefix=/usr/msp430'
+ # Note: binutils doesn't build without warnings on macOS
+ # https://sourceware.org/ml/binutils/2013-12/msg00051.html
+ configure_args_binutils=$(echo --disable-{sim,gdb,werror})
+ configure_args_gcc=$(echo --enable-target-optspace --enable-newlib-nano-formatted-io)
+ configure_args_gdb_common=$(echo --disable-{binutils,gas,ld,gprof,etc} --without-{mpfr,lzma} --with-static-standard-libraries --disable-source-highlight)
+ configure_args_gdb_nopy="--with-python=no"
+-configure_args_gdb_py="--with-python=python2.7 --program-prefix=msp430-elf- --program-suffix=-py"
+ 
+ pushd gcc
+   # Download sources for gcc prerequisites (gmp, mpfr, mpc, and isl)
+@@ -53,76 +52,35 @@
+   done
+ popd
+ 
+-# Download prerequisites for building MINGW GDB with python2.7 support.
+-if [[ "$configure_args_common" == *"mingw"* ]]; then
+-  python_ws=$(readlink -f mingw-python-workspace)
+-  rm -rf $python_ws
+-  mkdir $python_ws
+-  pushd $python_ws
+-    if [[ "$configure_args_common" == *"i686-w64-mingw32"* ]]; then
+-      python_win_msi_url=https://www.python.org/ftp/python/2.7.17/python-2.7.17.msi
+-    elif [[ "$configure_args_common" == *"x86_64-w64-mingw32"* ]]; then
+-      python_win_msi_url=https://www.python.org/ftp/python/2.7.17/python-2.7.17.amd64.msi
+-    else
+-      echo "ERROR: Unhandled value to --host configure option for python support"
+-    fi
+-    if [ ! -e ../$(basename $python_win_msi_url) ]; then
+-      curl -O $python_win_msi_url
+-    else
+-      cp ../$(basename $python_win_msi_url) .
+-    fi
+-    7za x $(basename $python_win_msi_url)
+-    # Used by python-config.sh
+-    export PYTHON_WIN=$python_ws
+-    configure_args_gdb_py="--with-python=$(readlink -f ../python-config.sh) --program-prefix=msp430-elf- --program-suffix=-py"
+-  popd
+-fi
+-
+ # Create directories
+ rm -rf build install
+ mkdir -p build/{binutils,gcc,gdb} install
+ 
+ # Build binutils
+ pushd build/binutils
+-  ../../binutils/configure $configure_args_common $configure_args_binutils --with-pkgversion="${pkg_version-$USER}"
+-  make
+-  make html
+-  make install install-html DESTDIR=$PWD/../../install
++  ../../binutils/configure $configure_args_common $configure_args_binutils
++  make -j
```

Comment:
is this right? `make` by itself was very slow. Maybe I could pass `makejobs` instead?